qualcoder/qualcoder.py

```diff
@@ -171,6 +171,30 @@ def load_settings(cls):
             f.close()
         return settings
 
+    def add_relations_table(self):
```

Comment:
What is the purpose of these tables? are they needed  for the view_graph?

Reply:
no it is not needed for graphviz, it is really an additional feature to create relations between codes which are not hierarchical like categories. An example  would be codeA <- contradicts -> codeB.

Reply:
ok, that makes sense